contexts/reducers/change-service-starting.ts

```diff
@@ -0,0 +1,19 @@
+import {ServiceState, State} from "../../interfaces/application-state";
+import {Action, SimpleActor} from "./reducer";
+
+export class ChangeServiceProp extends SimpleActor<boolean> implements Action<boolean> {
+
+  constructor(readonly id,
+              readonly prop: keyof ServiceState) {
+    super(id);
```

Comment:
if we add `AppStateIdThing.Service` to this we can then refer to `super.reduce()` and pass on `transformed` enabling us to use the base implementation with the debugging abilities